cmd/nodepool/kubevirt/create.go

```diff
@@ -291,6 +291,7 @@ func (o *KubevirtPlatformCreateOptions) NodePoolPlatform() *hyperv1.KubevirtNode
 	}
 
 	if o.Memory != "" {
+		// NOTE(mdbooth): this use of MustParse is unsafe, but this method cannot return an error
```

Comment:
We are looking to someway add debug traces changing the log framework, so we can add this as a debug trace when the times come. Could you please rephrase with something like :

// TODO: add a debug trace for this error